lib/jxl/encode_test.cc

```diff
@@ -1901,7 +1906,7 @@ TEST_P(EncoderStreamingTest, OutputCallback) {
   }
 
   EXPECT_TRUE(SameDecodedPixels(compressed, streaming_compressed));
-  EXPECT_LE(streaming_compressed.size(), compressed.size() + 16);
+  EXPECT_LE(streaming_compressed.size(), compressed.size() * 1.02 + 1024);
```

Comment:
is there some deeper reason behind this formula? I assume not, but...

Reply:
No deep reason, and I realized that the multiplier is not needed since these images are all small.